backend/database/connectDataBase.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const { Sequelize, DataTypes } = require('sequelize');
+const sequelize = new Sequelize(process.env.SEQUELIZE_URL, {logging: false});
+
+const connectDataBase = async () => {
+    sequelize
+        .authenticate()
+        .then((data) => {
+            console.log('Connecting with db successful...');
+        })
+        .catch((err) => {
+            console.log('Connecting ends with error...');
+        });
+        
+};
```

Comment:
Arrow function may be without { ... } body because of sequelize.authenticate().then() is single statement